tests/data/group_system/group_network/group_network-ipv6/group_configuring_ipv6/group_disabling_ipv6_autoconfig/rule_sysctl_net_ipv6_conf_all_accept_source_route/correct_value.pass.sh

```diff
@@ -7,3 +7,5 @@ if grep -q "^net.ipv6.conf.all.accept_source_route" /etc/sysctl.conf; then
 else
 	echo "net.ipv6.conf.all.accept_source_route = 0" >> /etc/sysctl.conf
 fi
+
```

Comment:
@ggbecker How do you run this test scenario on RHEL 8 target when this rule isn't part of RHEL 8 ospp profile and profile stig-rhel7-server-upstream doesn't exist in RHEL8 (and by the way doesn't exist anywhere)?

Reply:
I used the content from #4405 which introduces a new profile containing the rules tested here. I'm not sure how to proceed testing rules which are supposed to be included in CC effort but are not selected by any RHEL8 profile. When the test suite gets updated to support overriding `#profile` when running in rule+profile mode it might solve the issue (https://github.com/ComplianceAsCode/content/pull/4419#issuecomment-504375597).

Reply:
Basically, you're blocked by #4405 , because you need to add `ccc` to the `# profiles` line.

Reply:
> Basically, you're blocked by #4405 , because you need to add ccc to the # profiles line.

This is not true anymore because there will be only one profile.